src/resources/postcss/admin/category-colors/admin-category.pcss

```diff
@@ -0,0 +1,194 @@
+.table-view-list.tags {
```

Comment:
I've made suggestions throughout but:
use our props where you can - they should be available everywhere we load CSS.

margin and padding should use spacers
font size, weight, and line height use typography, etc.